packages/terra-props-table/src/PropsTable.scss

```diff
@@ -0,0 +1,13 @@
+:local {
+    .props-table-pre {
+        white-space: pre-wrap;
+    }
+
+    .prop-table-name, .prop-table-required, .prop-table-default {
+        width: 10%;
```

Comment:
If you add `vertical-align: top` it may make these easier to read with the type is really long. 
<img src="https://camo.githubusercontent.com/27a5abd31dd16bb009fdcbe434c1c442fd910778/68747470733a2f2f692e696d6775722e636f6d2f6f6d4473494e322e706e67">

It would align the text in the other columns to the top of their table cell.